test/test_load.cpp

```diff
@@ -0,0 +1,85 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#include <gmock/gmock-matchers.h>
+#include <gtest/gtest.h>
+
+#include <regex>
+
+#include <fusion.h>
+#include <inlining.h>
+#include <ir/utils.h>
+#include <ops/alias.h>
+#include <ops/arith.h>
+#include <ops/utils.h>
+#include <test/utils.h>
+#include <test/validator.h>
+#include <type.h>
+
+namespace nvfuser {
+
+class LoadTest : public NVFuserTest {};
+
+TEST_F(LoadTest, LoadCache) {
+  Fusion fusion;
+  FusionGuard fg(&fusion);
+
+  TensorView* tv0 = makeContigTensor(1);
+  fusion.addInput(tv0);
+  TensorView* tv1 =
+      ops::newValLike(tv0, tv0->getDataType().value())->as<TensorView>();
+  IrBuilder::create<LoadStoreOp>(
```

Comment:
I think you already understand this, but just to note that this is fine for testing, but in practice the caching option should be considered a scheduling parameter, so it shouldn't be set as part of the user fusion definition. Currently since we don't have a way to choose the caching option other than creating a `LoadStoreOp` this way, so makes sense.

Reply:
Thanks for the heads up, Naoya! I'll see how this goes with my future PRs. I may still keep this interface for easy testing.